src/main/java/com/iridium/iridiumskyblock/configs/Messages.java

```diff
@@ -23,6 +23,7 @@
     public String invitedPlayer = "%prefix% &7You have invited %player% to join your island";
     public String userInvitedPlayer = "%prefix% &7%inviter% has invited %player% to join your island";
     public String youHaveBeenInvited = "%prefix% &7%inviter% has invited you to join their island";
+    public String alreadyInYourIsland = "%prefix% &7This player is already in your island team";
```

Comment:
change to
     public String alreadyInYourIsland = "%prefix% &7This player is already in your island .";